src/TodoList.jsx

```diff
@@ -1,30 +1,15 @@
 import TodoListItem from "./TodoListItem";
 
-function TodoList() {
-    let todoList = [
-        {
-            id: 1,
-            title: "Laundry",
-        },
-        {
-            id: 2,
-            title: "Vacuum",
-        },
-        {
-            id: 3,
-            title: "Take out trash",
-        },
-    ];
-
-    return (
-        <>
-            <ul className="list">
-                {todoList.map((todo) => (
-                    <TodoListItem key={todo.id} item={todo}/>
-                ))}
-            </ul>
-        </>
-    );
+function TodoList({todoList}) {
+  return (
+    <>
```

Comment:
Nit: You don't need this surround React fragment since you just have one direct child (the `ul`)

Reply:
ok thank you! will update on the next PR. :)